extension/scripts/global/globalData.js

```diff
@@ -214,6 +215,7 @@ const DEFAULT_STORAGE = {
 				highlights: new DefaultSetting({ type: "array", defaultValue: [{ name: "$player", color: "#7ca900" }] }),
 				titleHighlights: new DefaultSetting({ type: "array", defaultValue: [] }),
 			},
+			casino: {},

```

Comment:
For an object as type you need to put `new DefaultSetting({ type: "object", defaultValue: {} })`

Reply:
I actually don't need that so I left it as a place for future options. Removed it.